models/account.py

```diff
@@ -203,57 +207,58 @@ def get_magic_link(
         :param round_id: The round id
         :return: True if successfully created
         """
-        new_account = False
         account = cls.get_account(email)
         if not account:
-            account = new_account = cls.create_account(email)
+            account = cls.create_account(email)
         if account:
-
-            fund = FundMethods.get_fund(fund_id)
-            round_for_fund = get_round_data(
-                fund_id=fund_id, round_id=round_id, as_dict=True
-            )
-
-            notification_content = {
-                NotifyConstants.MAGIC_LINK_REQUEST_NEW_LINK_URL_FIELD: Config.AUTHENTICATOR_HOST  # noqa
-                + Config.NEW_LINK_ENDPOINT,
-                NotifyConstants.MAGIC_LINK_CONTACT_HELP_EMAIL_FIELD: round_for_fund.contact_details[  # noqa
-                    "email_address"
-                ],  # noqa
-                NotifyConstants.MAGIC_LINK_FUND_NAME_FIELD: fund.name,
-            }
-            if (
-                fund_id
-                and round_id
-                and new_account
-                and Config.CREATE_APPLICATION_ON_ACCOUNT_CREATION
-            ):
-                current_app.logger.info(
-                    "Preparing to auto-create a blank application for"
-                    f" account: {str(account)}, for fund_id: {fund_id}and"
-                    f" round_id: {round_id}"
+            if fund_short_name and round_short_name:
```

Comment:
Do we need 2 if branches for this, couldn't we have the notification stuff on the same if/else branch?  Not a big deal though, think this may be more readable the way you've done it actually

Reply:
this will also get chopped/ neatened down eventually once R2W3 closes 👌🏾